bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx

```diff
@@ -20,17 +20,60 @@ categories:
   - You have [created a Mac mini M1](/bare-metal/apple-silicon/how-to/create-mac-mini-m1/)
 </Message>
 
-VNC (Virtual Network Computing) is a desktop-sharing system that allows users to remotely access and control a computer's desktop from another device. Scaleway users renting a Mac mini M1 can use VNC by simply enabling the "Screen Sharing" or "Remote Management" configuration on their Mac mini's system preferences. Using the Mac mini's IP address or hostname, a user can then connect to the M1 from another device through a VNC client (such as RealVNC, TightVNC, etc.).
 
-However, some users that have recently upgraded their macOS are currently encountering a bug preventing them from effectively using the screen sharing feature. Error messages include `Connection failed to <M1s_address>` and `Make sure Screen Sharing or Remote Management (in the Sharing section of System Preferences) is enabled on the remote computer`.
+VNC (Virtual Network Computing) is a system that lets users remotely access and control another computer's desktop. For those using Scaleway's rented Mac mini M1, VNC can be activated by turning on either the "Screen Sharing" or "Remote Management" options in the system preferences. Once set up, you can connect to the Mac mini M1 from another device using its IP address or hostname through VNC clients like RealVNC, TightVNC, and others.
+
+However, some users who've updated their macOS have faced issues with this feature, receiving error messages like `Connection failed to <M1s_address>` and `Make sure Screen Sharing or Remote Management is enabled on the remote computer` under the Sharing section of System Preferences. 
+
+Upon further analysis, we found a design flaw in macOS 13 and 14, where brute-force attacks performed by malicious actors could result in the service halting. [Find out more about this issue on our blog](https://www.scaleway.com/en/blog/m1-how-a-design-flaw-slowed-down-apples-vnc/).
```

Comment:
let's talks about macOS in general, no need to tell about the version, we don't support macOS 14 yet btw